pkg/opsmanager/api_set_project_tags.go

```diff
@@ -0,0 +1,30 @@
+package opsmanager
+
+import (
+	"bytes"
+	"encoding/json"
+
+	"github.com/mongodb-labs/pcgc/pkg/useful"
+)
+
+func (client opsManagerClient) SetProjectTags(projectID string, tags []string) (ProjectResponse, error) {
+	var result ProjectResponse
+
+	url := client.resolver.Of("/groups/%s", projectID)
+
+	request := struct {
```

Comment:
I'd avoid inline structs and keep thing consistent (define structs at the top of the file, or maybe in a totally separate file in the package)